src/NHibernate/Logging.cs

```diff
@@ -126,11 +143,10 @@ public interface INHibernateLoggerFactory
 	/// By default, it will use a <see cref="Log4NetLoggerFactory"/> if log4net is available, otherwise it will
 	/// use a <see cref="NoLoggingLoggerFactory"/>.
 	/// </remarks>
-	public class LoggerProvider
+	public static class LoggerProvider
```

Comment:
Switched to static: constructor was private (so not a breaking change) and it simplifies the implementation.